### PR TITLE
Add bare-bones autoflasher guide

### LINKED ISSUE #15: Autoflash guide
How to use the autoflasher

### PR DESCRIPTION
**Pull request by sigvef**
This fixes #15.

### CODE REVIEW

File: guides/autoflash.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+layout: guide
+---
+
+# Autoflashing
+
+The mbed ecosystem makes it really easy and convenient to compile and flash your development board, but manually downloading and copying the compiled hex file can get really tedious after a while.
+To remedy this, we have developed a small python script that watches your current dlownload directory for new compiler downloads, and and automatically flashes the mbed for you.
```
Comment (stianjensen) [inline guides/autoflash.md]: dlownload

Comment (stianjensen) [inline guides/autoflash.md]: and and

**Comment by sigvef** [comment]
ping

**Comment by cristeahub** [comment]
I would like for a bit more information about how to actually use it, like a command line example

**Pull request merged**